torchtext/csrc/vocab.cpp

```diff
@@ -77,6 +91,11 @@ void Vocab::insert_token(const std::string &token, const int64_t &index) {
 
 std::string Vocab::lookup_token(const int64_t &index) {
   if (index < 0 || index > static_cast<int64_t>(itos_.size())) {
+#ifdef _MSC_VER
```

Comment:
Just for the record, we need further investigation (probably some help from MS) on the propagation of RuntimeError through pybind.

Reply:
Yup added a issue to the PR description to keep track of this https://github.com/pytorch/text/issues/933